OrbitClientGgp/ClientGgp.cpp

```diff
@@ -275,8 +275,8 @@ void ClientGgp::OnCaptureStarted(
 
 void ClientGgp::OnCaptureComplete() {
   LOG("Capture completed");
```

Comment:
Not related with this PR but just wondering comparing with the code in App if it is missing `GetMutableCaptureData().FilterBrokenCallstacks();` here? I don't think it is a big deal but just wondering if it should be added to the client as well.

Reply:
I would say, yes, it should be added here. Otherwise, we have an inconsistency in captures taken with this client vs the UI client.

Reply:
(But would say it's out-of-scope for the PR. Will you take care of adding it?)

Reply:
Yes, I will add it to my todo list. Thanks!

Reply:
Thank you!